Doc/library/dis.rst

```diff
@@ -210,6 +213,11 @@ operation is being performed, so the intermediate analysis object isn't useful:
    This generator function uses the ``co_firstlineno`` and ``co_lnotab``
    attributes of the code object *code* to find the offsets which are starts of
    lines in the source code.  They are generated as ``(offset, lineno)`` pairs.
+   See :source:`Objects/lnotab_notes.txt` for the ``co_lnotab`` format and
+   how to decode it.
+
+   .. versionchanged:: 3.6
+      Added support for negative line number delta.
```

Comment:
This looks as implementation detail that should be described in `lnotab_notes.txt`. The change visible for the user of `findlinestarts()` is that line numbers now can not be monotonically increasing.